### PR TITLE
Feature/movie detail view

### PR DESCRIPTION
**Pull request by rafu07**
## Definition of Done

- Acceptance criteria for the user story / bug / features etc. are met
- All changes which go beyond the acceptance criteria should be created in a new ticket
- All non-functional requirements are met
- Unit tests must be written for each feature, so that the test-coverage is met (60%)
- All unit tests must have been passed successfully
- For a pull request in Dev, at least one person must give an approval (code review)
- After the changes are in the dev they have to be inspected again

### All Submissions:

* [ ] Have you followed the guidelines of the definiton of done?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

**The new feature includes:**

Enthält die Implementierung für das Movie DetailPage Feature im Frontend 

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: src/main/java/com/kinoticket/backend/rest/MoviesController.java [hunk 1]
```diff
@@ -5,13 +5,9 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
+@CrossOrigin("*")
```
Comment (niklasweidenfeller) [inline src/main/java/com/kinoticket/backend/rest/MoviesController.java]: Notwendig?

File: src/main/java/com/kinoticket/backend/rest/ImageController.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.annotation.Resource;
+import java.io.IOException;
+import java.io.InputStream;
+
+
+@RestController
+@CrossOrigin("*")
```
Comment (niklasweidenfeller) [inline src/main/java/com/kinoticket/backend/rest/ImageController.java]: Notwendig?

**Review by niklasweidenfeller — APPROVED** [review]
Sieht gut aus, nur zwei Anmerkungen

File: src/main/java/com/kinoticket/backend/rest/ImageController.java (hunk 2 above)
Comment (rafu07) [reply src/main/java/com/kinoticket/backend/rest/ImageController.java]: Hmm das weiß ich leider nicht. Eigentlich wird das ja init durch beim Start der Application gemacht. Leider hat das bei mir nicht funktioniert, sodass ich das für die Controller extra machen musste. Kann aber auch nur so ne Mac Sache sein. Also ich würde es dann einfach entfernen 👍

Comment (niklasweidenfeller) [reply src/main/java/com/kinoticket/backend/rest/ImageController.java]: Läuft das Frontend auf Mac auch unter Port 3000?

**Pull request merged**